### PR TITLE
how-to-use-fs-create-write-stream pt-br

### PR DESCRIPTION
**Pull request by VacariGabriel**

### CODE REVIEW

File: locale/pt-br/knowledge/advanced/streams/how-to-use-fs-create-write-stream.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: Como usar fs.createWriteStream?
```
Comment (TiagoDanin) [inline locale/pt-br/knowledge/advanced/streams/how-to-use-fs-create-write-stream.md]: Sugestion: `Como usar o fs.createWriteStream?`

File: locale/pt-br/knowledge/advanced/streams/how-to-use-fs-create-write-stream.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+date: '2011-08-26T10:08:50.000Z'
+tags:
+  - core
+  - streams
+  - fs
+difficulty: 3
+layout: knowledge-post.hbs
+---
+
+A função `fs.createWriteStream()` cria um fluxo corrente de uma maneira muito simples. Depois de chamar a função `fs.createWriteStream` com o caminho da pasta, voce tem um fluxo corrente. Os objetos de respostas (assim como a solicitação) são fluxos. Então vamos transmitir os dados `POST` para o arquivo `output`. Como o código é simples, é muito fácil lê-lo e comentar por que cada linha é necessária.
```
Comment (TiagoDanin) [inline locale/pt-br/knowledge/advanced/streams/how-to-use-fs-create-write-stream.md]: `voce` ~> `você`

**Review by TiagoDanin — CHANGES_REQUESTED** [review]

**Review by TiagoDanin — APPROVED** [review]
:100: LGTM

**Review by glaucia86 — APPROVED** [review]
LGTM 👍 

Thank you for your contribution @VacariGabriel

**Comment by glaucia86** [comment]
cc: @Maledong @Trott approved it this translation pt-br

**Review by aymen94 — APPROVED** [review]

**Comment by ZYSzys** [comment]
@VacariGabriel Thanks for contributing!

**Pull request merged**